nginx/frontend

```diff
@@ -4,6 +4,11 @@ server {
     location ~ ^(?:/[-a-z/]+)?/static/(.+)$ {
         alias /app/app/static/$1;
 
+        gzip on;
+        gzip_disable "msie6";
+        gzip_types text/css application/javascript;
```

Comment:
I know this is very minor but could we gzip our favicon? `image/x-icon` it was flagged up in recent performance test. 

🤔  I know we don't have many images but maybe its worth adding gzip to images in general

Reply:
Most image file formats are compressed internally and there won't really be any benefit to extra gzipping. It can even make them bigger in some cases (!). But if our favicon is a very old-skool `.ico` it'll be a completely uncompressed bitmap, so it might be worth it - I'll check.

Reply:
Yup gzipping takes it from 4.5k -> 1.7k, so I'll add that. It requires a bit of cooperation from the browser too though - it's possible the browser won't supply an appropriate `Accept-Encoding: gzip` header when requesting images, dunno...